rust/lance/src/dataset.rs

```diff
@@ -120,11 +121,6 @@ impl From<&Manifest> for Version {
 
 /// Customize read behavior of a dataset.
 pub struct ReadParams {
-    /// The block size passed to the underlying Object Store reader.
-    ///
-    /// This is used to control the minimal request size.
-    pub block_size: Option<usize>,
-
```

Comment:
This was redundant, since `block_size` is a field in `ObjectStoreParams` (another field in this struct).